ThreeDSecure/build.gradle

```diff
@@ -30,6 +30,7 @@ android {
 
 dependencies {
     api project(':BraintreeCore')
+    api project(':Card')
```

Comment:
Does this mean a merchant using 3DS, also doesn't need to add `implementation 'com.braintreepayments.api:card:4.0.0-beta1'` to their gradle?

Reply:
Yup! Merchant can still add card to their gradle, but they aren't required to do so. Gradle will always pick the newest version.

Reply:
Awesome!